### ISSUE TITLE
copy formatting option in ckeditor

### ISSUE THREAD

**Issue by gerald2545** [znuny/Znuny#370]
### Environment

- OS:           windows 10 
- Browser:      firefox, chrome
- Znuny version: 6.5.1

### Expected behaviour

have the option in ckeditor toolbar to copy formatting
![2023-04-20_172250](https://user-images.githubusercontent.com/10177830/233412898-9b200ff2-5050-44e9-a7ba-617d62859d6a.png)

### Actual behaviour

no icon is available for this option

### Additional information

it's not really a problem, but asked by my colleagues. I used to add it for OTRS....
with command line
vi /opt/otrs/Kernel/Output/HTML/Layout.pm
:%s/'Bold'/'CopyFormatting', 'Bold'/

if you think it can be integrated in znuny source code, I can suggest a merge request

**Comment by hanneshal** [comment]
Hi @gerald2545 
I guess this is meant to be a feature request? I can't remember to have this feature in the past.

If you know what the plugin is called and checked that it is delivered with Znuny, 
you can just add in the sysconfig. 

```
Frontend::RichText::Settings###ExtraPlugins
```

Regards
Johannes

**Comment by gerald2545** [comment]
yes, it's a feature request...I hope I didn't post at the wrong plage...will check.
I saw in the release notes that ckeditor could be configured in "system configuration"..but didn't find help in znuny documentation
I just test : 
add copyformatting to 
Frontend::RichText::Settings###ExtraPlugins and deployed
![image](https://user-images.githubusercontent.com/10177830/233419833-a1a7648e-d20f-4e9a-8e03-2ef8605d33ff.png)

I have no new icon. I even did 
```
otrs@esitoul-rocky-prod: ~/bin $ ./otrs.Console.pl Maint::Cache::Delete
Deleting cache...
Done.

otrs@esitoul-rocky-prod: ~/bin $ ./otrs.Console.pl Maint::Loader::CacheCleanup
Cleaning up the loader cache files...
Done.

otrs@esitoul-rocky-prod: ~/bin $ ./otrs.Console.pl Maint::Loader::CacheGenerate
Generating loader cache files...
Done.
```
but no changes in ckeditor toolbar

I think it is deleverd with znuny as if modify /opt/otrs/Kernel/Output/HTML/Layout.pm
with
:%s/'Bold'/'CopyFormatting', 'Bold'/

it works

**Comment by gerald2545** [comment]
i find the plugin code on https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_plugins_copyformatting.html

**Comment by hanneshal** [comment]
I checked it for you. It part of the plugin list and you can add it.
<img width="525" alt="image" src="https://user-images.githubusercontent.com/804939/233443923-1902c89c-0d41-488c-8e39-8464b626f5ef.png">

**Comment by rkaldung** [comment]
Since the question is answered we'll close this issue.
